cmd/bffutil/main.go

```diff
@@ -1089,6 +1102,50 @@ func sortAppdataOrgs(c *cli.Context) (err error) {
 	return nil
 }
 
+func dedupeAppdataOrgs(c *cli.Context) (err error) {
+	// Get all users in the tenant
+	var users *management.UserList
+	if users, err = auth0.User.List(); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	// Recreate org list for each user
+	for _, user := range users.Users {
+		appdata := &auth.AppMetadata{}
+		if err = appdata.Load(user.AppMetadata); err != nil {
+			return cli.Exit(err, 1)
+		}
+
+		// Get the user's current org list
+		orgs := appdata.GetOrganizations()
+		if len(orgs) == 0 {
+			continue
+		}
+
+		// Check for duplicate orgs and remove them.
+		seen := make(map[string]bool)
```

Comment:
One trick I usually use for things like this is to use an empty struct instead of the bool.

```
seen := make(map[string]struct{})
```

Then instead of `seen[org] = true`, you just do `seen[org] = struct{}{}`. It doesn't matter much since this is a script, but using the struct instead of the bool actually saves some memory and the existence checks will work the same way (`struct{}{}` doesn't take up any space but `bool` does).

Reply:
That's great to know. Thanks @pdeziel!